dotCMS/src/main/java/com/dotcms/rest/api/v1/storage/MetadataForm.java

```diff
@@ -0,0 +1,90 @@
+package com.dotcms.rest.api.v1.storage;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = MetadataForm.Builder.class)
+public class MetadataForm {
+
+    private final String identifier;
+    private final String inode;
+    private final String language;
+    private final String field;
+    private final boolean nocache;
+
+    public MetadataForm(final Builder builder) {
+
+        this.identifier = builder.identifier;
+        this.inode      = builder.inode;
+        this.language   = builder.language;
+        this.field      = builder.field;
+        this.nocache    = builder.nocache;
+    }
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public String getInode() {
+        return inode;
+    }
+
+    public String getLanguage() {
+        return language;
+    }
+
+    public String getField() {
+        return field;
+    }
+
+    public boolean isNocache() {
+        return nocache;
+    }
+
+    @Override
+    public String toString() {
+        return "MetadataForm{" +
+                "identifier='" + identifier + '\'' +
+                ", inode='" + inode + '\'' +
+                ", language='" + language + '\'' +
+                '}';
+    }
+
+    public static final class Builder {
+
+        @JsonProperty private String identifier;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field identifier has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)